server/repository/migrations/postgres/2022-10-11T100-22_v102/up.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TYPE key_type
```

Comment:
🤔  I expect concerns about this.. it is helpful for me though. I didn't have to recreate my test database